lib/internal/url.js

```diff
@@ -23,6 +25,19 @@ const IteratorPrototype = Object.getPrototypeOf(
   Object.getPrototypeOf([][Symbol.iterator]())
 );
 
+const usvRe =
+    /([^\uD800-\uDBFF]|^)[\uDC00-\uDFFF]|[\uD800-\uDBFF](?![\uDC00-\uDFFF])/;
+function toUSVString(val) {
+  if (typeof val === 'symbol')
+    throw new TypeError('Cannot convert a Symbol value to a string');
+  const str = String(val);
```

Comment:
I'd just `val = '' + val` if you want to coerce, that way you don't have to duplicate the 'is symbol?' check.